packages/runtime/container-runtime/src/orderedClientElection.ts

```diff
@@ -206,8 +208,10 @@ export interface IOrderedClientElectionEvents extends IEvent {
 export interface ISerializedElection {
     /** Sequence number at the time of the latest election. */
     readonly electionSequenceNumber: number;
-    /** Most recently elected client id. */
+    /** Most recently elected client id. May be interactive client or summarizer client. */
```

Comment:
May be worth adding that this can be an interactive client only if its same as the parent client and its an indication that a summarizer client has to be spawned.

Reply:
Thanks. That's a good suggestion.